regolith/templates/resume.tex

```diff
@@ -208,13 +208,13 @@
       {% if 'narrative' in grant or 'benefit_of_collaboration' in grant or 'call_for_proposals' in grant %}
         \resitem{
           {%- if 'narrative' in grant %}
-            \href{ {{-latex_safe(grant['narrative'])-}} }{Narrative}.
+            \href{ {{-latex_safe(grant['narrative'], url_check=False)-}} }{Narrative}.
           {%- endif %}
           {%- if 'benefit_of_collaboration' in grant %}
-            \href{ {{-latex_safe(grant['benefit_of_collaboration'])-}} }{Benefit of Collaboration}.
+            \href{ {{-latex_safe(grant['benefit_of_collaboration'], url_check=False)-}} }{Benefit of Collaboration}.
```

Comment:
Maybe we should remove latex_safe from things that are URLs outright?

Reply:
yeah, probably